keyboards/cutie_club/borsdorf/config.h

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020 Cutie Club
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFB9C
+#define PRODUCT_ID 0x6D8A
+#define DEVICE_VER 0x0001
+#define MANUFACTURER Cutie Club
+#define PRODUCT Borsdorf
+#define DESCRIPTION Borsdorf keyboard
```

Comment:
```suggestion#
```

Reply:
What is the problem?

Reply:
sorry, stray char in the suggestion markdown

Reply:
No problem! Thank you for response